src/main/java/org/pac4j/demo/dw/factory/DropwizardConfigFactory.java

```diff
@@ -0,0 +1,10 @@
+package org.pac4j.demo.dw.factory;
+
+import org.pac4j.core.config.ConfigFactory;
+
+import java.util.Map;
+
+public interface DropwizardConfigFactory extends ConfigFactory {
```

Comment:
We should, but in fact, we won't even need it. We'll do better by changing the `ConfigFactory` signature for the `build` method from `build()` to `build(Object... parameters)` to be able to pass the properties we want.